### DISCUSSION TITLE
repository location

### DISCUSSION THREAD

**Discussion by beliaev-maksim** [ansys/aedt-testing#233] [General]
@akaszynski I would kinda disagree that this tool is internal only. (see org)

the problem that it is not adopted by the community does not make it internal. The whole purpose is to provide customers this framework

cc: @boyang2022 @Samuelopez-ansys

**Comment by akaszynski** [comment]
@beliaev-maksim, public repositories are likely being moved to the https://github.com/ansys, which seems to be where open source repositories are going.

**Reply by beliaev-maksim** [reply]
Then the purpose of internal org is not clear :)

There are indeed some tools that are interal but can be open source, in case any customer will want such functionality without official support

Just an opinion :)

**Reply by Samuelopez-ansys** [reply]
@beliaev-maksim , I agree this repo should be in Ansys org., but all repositories have been moved to ansys-internal, and are under review. This one will be transferred to ansys org. in the future as @akaszynski said, because it is public and does not contain any IP.
